drivers/cleanup.py

```diff
@@ -2741,8 +2760,8 @@ def get_state(srUuid):
     locking.
     """
     init(srUuid)
-    if lockRunning.acquireNoblock():
-        lockRunning.release()
+    if lockActive.acquireNoblock():
```

Comment:
Spawned, this was in the patch queue but missed from the push. This is what caused the BVT to fail in the first submission.